### PR TITLE
Enable email reports in prod

### PR DESCRIPTION
**Pull request by apereverzin**
### Change description ###

https://github.com/hmcts/cnp-flux-config/pull/3014

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/reform-scan-blob-router/values.yaml [hunk 1]
```diff
@@ -41,8 +41,8 @@ java:
 
     BULK_SCAN_PROCESSOR_URL: http://bulk-scan-processor-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
 
-    SMTP_HOST: "false"
-    SEND_DAILY_REPORT_ENABLED: "false"
+    SEND_DAILY_REPORT_ENABLED: "true"
```
Comment (doncem) [inline charts/reform-scan-blob-router/values.yaml:44]: if prod only then need to go into flux config instead

Comment (apereverzin) [reply charts/reform-scan-blob-router/values.yaml:44]: and SMTP_HOST - also to flux?

File: charts/reform-scan-blob-router/values.aat.template.yaml [hunk 2]
```diff
@@ -4,6 +4,7 @@ java:
     STORAGE_BLOB_PROCESSING_DELAY_IN_MINUTES: 0
     DELETE_DISPATCHED_FILES_CRON: "0/10 * * * * *"
     SEND_NOTIFICATIONS_CRON: "0/10 * * * * *"
+    SEND_DAILY_REPORT_ENABLED: "false"
```
Comment (nitinprabhu) [inline charts/reform-scan-blob-router/values.aat.template.yaml:7]: we should not have any values here

Comment (apereverzin) [reply charts/reform-scan-blob-router/values.aat.template.yaml:7]: yes, as Don said - changes will be only in flux config

File: charts/reform-scan-blob-router/values.yaml (hunk 1 above)
Comment (doncem) [reply charts/reform-scan-blob-router/values.yaml:44]: microsoft provides that value openly so it is a nobrainer to google the value

Comment (apereverzin) [reply charts/reform-scan-blob-router/values.yaml:44]: I will close this PR

**Comment by apereverzin** [comment]
https://github.com/hmcts/cnp-flux-config/pull/3014

**Pull request closed without merging**